### PR TITLE
Use new syntax for applying plugins on subprojects

### PR DESCRIPTION
**Pull request by loosebazooka**
https://docs.gradle.org/3.4/userguide/plugins.html#sec:subprojects_plugins_dsl

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -17,17 +17,16 @@
 buildscript {
     repositories {
         maven { url "https://maven-central.storage.googleapis.com"}
-        maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath 'gradle.plugin.org.jetbrains:gradle-intellij-plugin:0.1.10'
         classpath "net.ltgt.gradle:gradle-errorprone-plugin:0.0.8"
```
Comment (nkibler) [inline build.gradle]: Can we do this for errorprone too?

Comment (loosebazooka) [inline build.gradle]: I don't think errorprone is coming from the gradle plugin repository (https://plugins.gradle.org/) in this case

Comment (loosebazooka) [inline build.gradle]: actually I just looked it up and it can come from there

Comment (loosebazooka) [inline build.gradle]: lemme update the build and check.

**Comment by loosebazooka** [comment]
So there's something in there which I'm not super clear on, the inclusion of the google maven repository mirror. I wonder if kokoro is slower or unaffected without that in there.

**Comment by nkibler** [comment]
It was introduced in #1406 under the assumption that it does speed up builds; maybe @patflynn knows more?

**Comment by patflynn** [comment]
When I tested it out I did see a decent speedup in the build times.
 alexsloan saw the opposite.

**Comment by patflynn** [comment]
I'm still inclined to keep it though because gcs is less flaky than maven central

**Comment by patflynn** [comment]
@loosebazooka we merging this?

**Comment by loosebazooka** [comment]
I guess not, I forgot I wrote this. It's up to you guys really, if you want to.

File: build.gradle [hunk 2]
```diff
@@ -19,16 +19,13 @@ buildscript {
         maven { url "https://maven-central.storage.googleapis.com"}
```
Comment (loosebazooka) [inline build.gradle]: This CL needs some changes if you still plan on merging, these lines are probably unnecessary, gradle pulls in all plugin dependencies when using the new format from jCenter. I would probably just rewrite it if you guys wanted to actually do this.

**Pull request closed without merging**